wasp/src/main/java/com/orhanobut/wasp/MethodInfo.java

```diff
@@ -85,7 +85,7 @@ private void parseMethodAnnotations() {
 
             if (annotationType == Mock.class) {
                 Mock mock = (Mock) annotation;
-                mockType = mock.value();
+                this.mock = new WaspMock(mock.statusCode(), mock.path());
```

Comment:
Maybe we can put a file exists check here, if the path is not empty,null, we can check if it really exists, if not exists regarding to conditions, would be nice to throw null pointer exception to warn the developer.